proto/cosmos/staking/v1beta1/query.proto

```diff
@@ -365,7 +365,7 @@ message QueryHistoricalInfoRequest {
 // method.
 message QueryHistoricalInfoResponse {
   // hist defines the historical info at the given height.
-  HistoricalInfo hist = 1;
+  Historical hist = 1;
```

Comment:
is this a valid way of doing it @testinginprod

Reply:
i can introduce it as a new field. it will help in avoiding this. Ill make the change